src/components/shared/user-courses/share/progress-status.scss

```diff
@@ -0,0 +1,17 @@
+$progress-bar-color: #000;
+
+.progress-bar-wrapper {
+    position: relative;
+    width: 280px;
+
+    .progress-bar-label {
+        position: absolute;
+        left: 0;
+        top: 0;
+        font-size: .75rem;
+        margin-left: 8px;
+        color: #000;
```

Comment:
according to Figma when a label has value the color is white otherwise it's black am I wrong?

Reply:
You are right. but in this particular case the color must be blackish one. not sure of the exact color hex number but it is dark